src/sql-parser/src/ast/mod.rs

```diff
@@ -600,14 +601,21 @@ impl Default for Envelope {
 
 impl fmt::Display for Envelope {
     fn fmt(&self, f: &mut fmt::Formatter) -> fmt::Result {
-        write!(
-            f,
-            "{}",
-            match self {
-                Self::None => "NONE", // this is unreachable as long as the default is None, but include it in case we ever change that
-                Self::Debezium => "DEBEZIUM",
+        match self {
+            Self::None => {
+                write!(f, "NONE")?;
+            } // this is unreachable as long as the default is None, but include it in case we ever change that
```

Comment:
Can you move this comment above the `Self::None` block? It now looks like it is applying to `Self::Debezium`.